gtk-2.0/gtkrc

```diff
@@ -168,8 +168,8 @@ style "murrine-thin" {
 # Notebook
 
 style "clearlooks-notebook-bg" {
-	bg[NORMAL] = @bg_color
-	bg[ACTIVE] = shade (0.80, @bg_color)
+	bg[NORMAL] = @base_color
+	bg[ACTIVE] = shade (0.87, @base_color)
```

Comment:
the idea to use base_color for notebook bg not seems logical to me -- it's usually used for user input, so it will be better to revert the change